src/main/java/koreatech/in/service/OwnerServiceImpl.java

```diff
@@ -83,7 +95,9 @@ public void requestVerification(VerifyEmailRequest verifyEmailRequest) {
 
         emailAddress.validateSendable();
 
-        putRedisFor(emailAddress.getEmailAddress(), ownerInVerification);
+        String email = emailAddress.getEmailAddress();
+
+        putRedisFor(StringRedisUtilObj.makeOwnerKeyFor(email), ownerInVerification);
```

Comment:
```suggestion
        putRedisFor(StringRedisUtilObj.makeOwnerKeyFor(emailAddress.getEmailAddress()), ownerInVerification);
```
String email이 한 번만 사용된다면 굳이 변수로 추출하지 않고 이런식으로 쓸 수 있을 것 같아요!

Reply:
넵 수정하겠습니다!

Reply:
그런 의도가 있으면 굳이 분리하진 않아도 될 것 같아요
역할도 같은 역할이고...


다만
1. `StringRedisUtilObj` 에서 키를 가져온다.
2. OwnerService에서 get(read), put(write)를 진행한다.

로 이해했는데

- redismapper에서 상수로 된 키를 enum으로 변경 -> 역할 분리
- 별도 클래스 RedisOwnerMapper(예시) 에서 enum을 인자로해서 read, write 하는 메서드 만들기 -> 역할분리
은 어떨까요..?